forge/db/models/AccessToken.js

```diff
@@ -29,18 +29,35 @@ module.exports = {
             }
         },
         ownerId: { type: DataTypes.STRING },
-        ownerType: { type: DataTypes.STRING }
+        ownerType: { type: DataTypes.STRING },
+        refreshToken: {
+            type: DataTypes.STRING,
+            set (value) {
+                if (typeof value === 'string') {
+                    this.setDataValue('refreshToken', sha256(value))
+                }
+            }
+        }
     },
```

Comment:
I'm not sure what the if block actually does here?

Reply:
If `refreshToken` is `null` then we shouldn't try to hash it.